docs/USAGE-GUIDE.md

```diff
@@ -1585,6 +1585,102 @@ int s2n_config_add_ticket_crypto_key(struct s2n_config *config, const uint8_t *n
 **s2n_config_add_ticket_crypto_key** adds session ticket key on the server side. It would be ideal to add new keys after every (encrypt_decrypt_key_lifetime_in_nanos/2) nanos because
 this will allow for gradual and linear transition of a key from encrypt-decrypt state to decrypt-only state.
 
+## TLS1.3 Pre-Shared Key (PSK) Related Calls 
+
+S2N-TLS provides support for pre-shared keys that can be established out of band and pre-shared keys that can be established in a previous connection and then used to establish a new connection via session resumption for versions >= TLS1.3. The Pre-shared key is used to mutually authenticate the peers, under the assumption that it is not shared with others. This allows the parties to skip the certificate verification steps, saving bandwidth, communication rounds, and latency. When a PSK is used, the underlying assumption is that a party receiving a message that passes verification with the PSK knows that the message was sent by a party that also owns that PSK. 
```

Comment:
I prefer:
"S2N-TLS provides support for pre-shared keys established out-of-band. The pre-shared key is used to mutually authenticate the peers under the assumption the key is not shared with others. This allows parties to skip certificate verification, saving bandwidth, communication rounds, and latency. 

Why mention session resumption at all here? Also I'm not really sure what that last sentence is saying, it's a little unclear.